model/task_lifecycle.go

```diff
@@ -919,6 +919,13 @@ func UpdateBuildStatus(b *build.Build) (bool, error) {
 		return false, errors.Wrapf(err, "getting tasks in build '%s'", b.Id)
 	}
 
+	b.Aborted = false
+	for _, t := range buildTasks {
+		if t.Aborted {
+			b.Aborted = true
```

Comment:
We're only updating this locally it looks like; need to update in the DB as well (but only if the aborted state has actually changed I think)

Reply:
updating the db when changed